app/views/devise/sessions/new.html.haml

```diff
@@ -7,7 +7,7 @@
             Sign In
         .panel-body
           = semantic_form_for(resource, as: resource_name, url: session_path(resource_name)) do |f|
-            = f.input :login
+            = f.input :login, label: "Username / Email"
```

Comment:
I would expect you to have commented that in the description of the PR, as one of the things you are doing in this PR.